src/main/preferences/schema.json

```diff
@@ -34,6 +34,11 @@
     "type": "boolean",
     "default": false
   },
+  "autoCheck": {
+    "description": "General--Whether to automatically check related task.",
```

Comment:
I prefer put `autoCheck` in `Editor` part not the `General` part of setting.

Reply:
Thank you, I'll fix it.